### PR TITLE
DockerHub credentials in .travis.yml

### PR DESCRIPTION
**Pull request by nadade**
added DockerHub credentials (user sreapiary)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-      - ladislav@apiary.io
+    - ladislav@apiary.io
     on_success: change
     on_failure: always
-
+env:
+  global:
+  - secure: aYbsW7RjNFxay4hHRxebw/RVNvtJGrYiKZnVGELOTdOoJF4IR9LcC2CuStX3x6NP2PfzgAR8DaZu/ht4U11M/3JEGfjQm30E6AONEm29qdGvJhpEEYK5nGHTlWohDZRd23bUWn6+zLqUw6Nz/RGPmd/Yg57XUz6ZlIzFVBeKpaw=
+  - secure: GzMiwhqE584coOtMJCHVYhw7K3huasPrXPTxiyNzoH1nr/6DS2FRevWBMnO2BT+Nl4MHgJtzVz51VZORIHaV4Qbw7rhDmEx8bag6ZCn/fNFSmjkYOy4s9KuGK8KbAnZx8PD+mH4FMNy+8eQAC8CndmdP5f1FHer3ZowOL/YuZAc=
+  - secure: TF3938h3ZBSorKeJqltK9Zp35FDaPkFXmyWuyNfdWyQ8yQLPkblWubqLCf1bZ3g+QLJhu1i6j/hYY92DZCAc1ulTKsHAhv8c8TOSaycYn6hDCIltO1dh0KyoTp7zzBCq/YQ83IZ0G774R0RTxgode3G6cSr06kSGw/YDD6hkBnQ=
```
Comment (abtris) [inline .travis.yml]: why you have 3 lines with secure?

Comment (nadade) [inline .travis.yml]: @abtris DOCKER_PASS, DOCKER_USER, DOCKER_EMAIL; should I only encrypt the password?

Comment (abtris) [inline .travis.yml]: No, you can make all in one line.

Comment (abtris) [inline .travis.yml]: [You may add multiple entries to your .travis.yml with key “secure.” They will all be available to your program.](https://docs.travis-ci.com/user/encryption-keys#Usage)

Comment (abtris) [inline .travis.yml]: But this isn't blocking.

**Pull request merged**